__declspec(dllexport) does not imply extern: add it. This fixes compilation with gcc-10.1

--- a/ThirdParty/libxml2/vtklibxml2/include/libxml/xmlexports.h	2020-05-17 14:32:52.863037615 +0200
+++ b/ThirdParty/libxml2/vtklibxml2/include/libxml/xmlexports.h	2020-05-17 14:32:23.266063521 +0200
@@ -111,7 +111,7 @@
   #undef XMLCDECL
   #if defined(IN_LIBXML) && !defined(LIBXML_STATIC)
     #define XMLPUBFUN __declspec(dllexport)
-    #define XMLPUBVAR __declspec(dllexport)
+    #define XMLPUBVAR __declspec(dllexport) extern
   #else
     #define XMLPUBFUN
     #if !defined(LIBXML_STATIC)
